.buildkite/pipeline.yml

```diff
@@ -20,7 +20,7 @@ steps:
     command: "scripts/deploy.sh"
     concurrency: 1
     concurrency_group: "deploy"
-    branches: master
+    branches: fusion-monorepo-bot
```

Comment:
Hmm, not sure we want to land this change, because going forward we will want to deploy from master, correct?

Reply:
Right, this will be re-set to master once we're ready to commit to the new bot